lib/nylas/resources/events.rb

```diff
@@ -94,5 +94,18 @@ def send_rsvp(identifier:, event_id:, request_body:, query_params:)
         request_body: request_body
       )
     end
+
+    # Returns a list of recurring events, recurring event exceptions, and single events
+    # from the specifiedcalendar within a given time frame.
+    #
+    # @param identifier [String] Grant ID or email account to import events from.
+    # @param query_params [Hash] The query parameters to include in the request
+    # @return [(Array(Hash), String, String)] The list of events, API Request ID, and next cursor.
+    def events_import(identifier:, query_params:)
```

Comment:
```suggestion
    def list_event_imports(identifier:, query_params:)
```

I think we need to keep this more consistent with the rest of our APIs. Closest one I could find was scheduled send messages.